xrpl/helpers/structs.go

```diff
@@ -0,0 +1,24 @@
+package helpers
+
+import "reflect"
+
+// Function to convert struct to map[string]any
+func StructToMap(s interface{}) map[string]any {
```

Comment:
If we decide to pass a tx_blob there's no need to convert anything 👍🏻  so we can remove it, but good effort!